Extension/src/LanguageServer/client.ts

```diff
@@ -926,10 +926,23 @@ export class DefaultClient implements Client {
         paths.push(localize("noConfig.string", "Do not configure a compiler (not recommended)"));
         const index: number = await this.showSelectCompiler(paths);
         if (index === -1) {
+            settings.defaultCompiler = "";
+            if (showSecondPrompt) {
+                const setCompiler: string = localize("setCompiler.string", "Set Compiler");
+                const value: string | undefined = await vscode.window.showInformationMessage(localize("setCompiler.message", "You do not have a compiler configured. Unless you set your own configurations, IntelliSense may not be functional."), setCompiler);
```

Comment:
There's a repeat use of this string. For whichever string that gets used for all these 3 cases, could you try saving the string's value into a global variable and reference that variable instead of having several copies of the string?

Reply:
@browntarik It seems like the entire showInformationMessage itself could be made into a function for the cases when the messages are the same.